config.json

```diff
@@ -572,7 +572,13 @@
       "core": false,
       "difficulty": 6,
       "slug": "bowling",
-      "topics": null,
+      "topics":  [
+        "functions",
```

Comment:
This isn't a topic in the [topics list](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt) and I'm not sure if it's a useful topic for this exercise. How do you think it related to this exercise?

Reply:
there are two different operations 
roll and score so i added topic functions. I think it should be added to the topicslidt. Whats your say? @FridaTveit

Reply:
A lot of exercises will have multiple methods so I'm not sure if it's a very useful topic. Feel free to open a PR to [problem-specifications](https://github.com/exercism/problem-specifications) to add `functions` as a topic but either way I'm not sure it would be useful for this particular exercise. What do you think? :)